lib/mix/lib/mix/tasks/new.ex

```diff
@@ -264,6 +264,9 @@ defmodule Mix.Tasks.New do
   # The directory Mix will write compiled artifacts to.
   /_build/
 
+  # The directory generated by Elixir Language Server (ElixirLS).
```

Comment:
This directory depends on the development environment of each user. That's why I wouldn't include it by default here.

I would suggest to include that into a [`global .gitignore`](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).